config/locales/en.yml

```diff
@@ -19,6 +19,13 @@ en:
       not_confirmed: "A confirmation email was sent to your account at '%{email}'. You must follow the instructions in the email before your account can be activated"
     confirmations:
       invalid_token: "Invalid confirmation token. Please try again"
+      user_not_found: "Unable to find user with email '%{email}'."
+      sended: "An email has been sent to '%{email}' containing instructions for confirming your account."
+      send_instructions: "You will receive an email with instructions for how to confirm your email address in a few minutes."
+      send_paranoid_instructions: "If your email address exists in our database, you will receive an email with instructions for how to confirm your email address in a few minutes."
     mailer:
+      confirmation_instructions:
+        confirm_link_msg: "You can confirm your account email through the link below:"
+        confirm_account_link: "Confirm my account"
       unlock_instructions:
         account_lock_msg: "Your account has been locked due to an excessive number of unsuccessful sign in attempts."
```

Comment:
You'll see here I grabbed a few locales from some other underlying libraries. We can leave them but I think we should choose what to use. You'll also notice that `send_intructions:` and `send_paranoid_instructions:` serve the same purpose but allow the user to decide what they'd like to use. These two along with `sended:` all serve the same purpose but are worded differently.